### ISSUE TITLE
Re: FleatherField cursor does not stay visible when focused/keyboard is visible

### ISSUE THREAD

**Issue by Lomedhi** [fleather-editor/fleather#438]
Opening issue for visibility to this [issue #421](https://github.com/fleather-editor/fleather/issues/421)

RE: "Hi. I also want to give you master devs @Amir-P @amantoux some well deserved props because this package is amazing from where you took it and very sufficient for my use case. I also am having the same scrolling to the viewport issue with cursor going below the keyboard.

I did some research into the matter and it seems a similar issue was reported on the original repo project https://github.com/memspace/zefyr/issues/485. https://github.com/flutter/flutter/issues/10826

From the thread, there is some proposed solution with considering the offset with and without the keyboard, but I believe this is for pack https://github.com/singerdmx/flutter-quill/pull/111.

Here is the [implemented solution code](https://github.com/singerdmx/flutter-quill/commit/471c1cb7ee542977b777b0fc10cbaa5cfb35ddea).

I tried testing and debugging my own implementation (`Container`,`Column`, `SingleChildScrollView`,`Scaffold` different parent child code, expanded and/or scrollable as true/false, set and not setting maxHeight), but got the same result of a non scrolling past the scroll container but also not scrolling the cursor above the keyboard. I can leave code snippet, but if you wrap `FleatherField` in a `Listview` and then wrap listview in a `Container` is what the code would in essence be with other children in the listview as well."

## Steps to Reproduce

1. Build FleatherEditor/Field widget
2. Wrap FleatherField in Listview or scrollable widget
3. Wrap FleatherField in Container or non-scrolable widget

## Environment
- OS [iOS/Web/Android]
- Flutter version [3.24.2]
- Fleather version [1.9.0]

## Logs

```
```

**Comment by manudicri** [comment]
@Lomedhi Yeah actually right now I'm using Flutter Quill and got the same issue. I'll may try to fix by forking the package (if it is possible). That would be cool just to know how to get the cursor position coordinates

**Comment by amantoux** [comment]
@manudicri @Lomedhi will look it this issue next

**Comment by amantoux** [comment]
@Lomedhi can you test https://github.com/fleather-editor/fleather/tree/462-fleatherfield-cursor-goes-behind-the-keyboard-when-typing and let me know how it goes?
